### DISCUSSION TITLE
How to make jan reply more at one time?

### DISCUSSION THREAD

**Discussion by aidec** [janhq/jan#2309] [General]
The token has been set to 4096, and the number of words in the prompt word is also expected to be responded to, but the result is not as expected.

**Comment by lv333ming** [comment]
I'm asking this question too. 4096Token quantity is too small

**Comment by kalle07** [comment]
For me it depends on the model if I ask, sometimes the answer is 6 sentences and that's too long for me  ^^
maybe you can force it in system-prompt "give me a detailed explanation and elaboration"
